client-permissions/client.sh

```diff
@@ -0,0 +1,176 @@
+#!/bin/sh
+#
+# Environment variables used:
+#  - SERVER: hostname or IP-address of the NFS-server
+#  - EXPORT: NFS-export to test (should start with "/")
+
+# enable some more output
+set -x
+
+[ -n "${SERVER}" ]
```

Comment:
Without `set -e` this will not cause the script to exit. You will want to exit the script when these environment variables are not set.

Reply:
@nixpanic Done

Reply:
@nixpanic Done